src/map/attack.cpp

```diff
@@ -446,6 +446,26 @@ bool CAttack::CheckCounter()
     return m_isCountered;
 }
 
+bool CAttack::IsCovered()
+{
+    return m_isCovered;
+}
+
+bool CAttack::CheckCover()
+{
+    CBattleEntity* PCoverAbilityUser = m_attackRound->GetCoverAbilityUserEntity();
+    if (PCoverAbilityUser != nullptr && PCoverAbilityUser->isAlive())
+    {
+        m_isCovered = true;
+        m_victim = PCoverAbilityUser;
+    }
+    else {
```

Comment:
style guide specifies braces on newlines. not a huge deal doubt it woudl hold up the pr, but when you fix the spares this would be a quick edit